{{cookiecutter.project_slug}}/clients/mobile/react-native/README.md

```diff
@@ -8,14 +8,23 @@ This app is bootstrapped with the TN-Bootsrapper, it uses Expo as a wrapper fram
 
 After running the bootsrapper a mobile directory is created the following steps are needed to run and deploy the app
 
-## Running the app 
+## Running the app locally
 
 User `npm run start` to run the app and source the env variables 
 
 alternatively if you want to use expo run command 
 
 `source .env && SENTRY_PROJECT_NAME=${SENTRY_PROJECT_NAME} npx expo start`
 
+**When running the app locally and working against a local backend you will need to use a proxy**
+
+1. Download and install ngrok 
+2. Set up ngrok auth token (request an account from William Huster)
```

Comment:
subdomain is not working as expected, I think we just need to purchase an ngrok wildcard on a subdomain we can use
`devs.thinknimble.com`